indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
I think this will cause wreckage. There a lot of places that will be affected by this change.

Two things. 1) Why is the margin now positive? 2) Why is the margin in `em`? If we want not to use `px` for layout elements we should use `rem`, but never `em`.